dist/api_err.py

```diff
@@ -180,27 +206,32 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
 format_srcfile(tmp_file)
 compare_srcfile(tmp_file, '../src/conn/api_strerror.c')
 
-# Update the error documentation block.
-doc = '../src/docs/error-handling.dox'
-tmp_file = '__tmp_api_err' + str(os.getpid())
-tfile = open(tmp_file, 'w')
-skip = 0
-for line in open(doc, 'r'):
-    if not skip:
-        tfile.write(line)
-    if line.count('IGNORE_BUILT_BY_API_ERR_END'):
+def check_document_errors(tfile, skip, err_type, errors):
```

Comment:
We should have a better name for this function, I am assuming that the check here is to check `skip`? There're `tfile.write()` in the function, I don't have specific preference for the name but we should also know the function is doing write from its name.

Reply:
Fixed